src/nlsolve_powerflow.jl

```diff
@@ -64,84 +82,84 @@ function _dogleg!(p::Vector{Float64},
 end
 
 function _trust_region_step(time_step::Int,
-    cache::NLCache,
+    X::StateVectorCache,
     linSolveCache::KLULinSolveCache{Int32},
-    pf::PolarPowerFlow,
+    R::ACPowerFlowResidual,
```

Comment:
This change makes the code more confusing

Reply:
renamed to not have single-letter variable names